plugin/lens.vim

```diff
@@ -144,6 +149,10 @@ function! lens#win_enter() abort
     return
   endif
 
+  if index(g:lens#disabled_buftypes, &buftype != -1)
```

Comment:
Are you sure this is working? Seems like it should be:

`index(g:lens#disabled_buftypes, &buftype) != -1`